.github/workflows/patches.yml

```
@@ -70,12 +70,19 @@ jobs:
           repository: FFmpeg/FFmpeg
           ref: release/4.2
           path: ffmpeg-release-4.2
+      - name: Clone libaom
+        run: git clone https://aomedia.googlesource.com/aom
+      - name: Configure libaom
+        run: |
+          [ -d ~/.ccache ] && sudo chown -R "$USER": ~/.ccache
+          cmake -S aom -B aom/build-static -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=OFF -DENABLE_TESTS=0
+          sudo make -C aom/build-static -j "$(getconf _NPROCESSORS_ONLN 2>/dev/null || sysctl -n hw.ncpu)" install
```

```suggestion
          sudo -E make -C aom/build-static -j "$(getconf _NPROCESSORS_ONLN 2>/dev/null || sysctl -n hw.ncpu)" install
```

use `sudo -E` to make sure cache works and the cache goes to the correct directory